src/libraries/libraries-packages.proj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <TraversalGlobalProperties>BuildAllProjects=true</TraversalGlobalProperties>
     <AdditionalBuildTargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">$(AdditionalBuildTargetFrameworks);package-$(Configuration)</AdditionalBuildTargetFrameworks>
+    <BuildAllOOBPackages Condition="'$(BuildAllOOBPackages)' == ''">true</BuildAllOOBPackages>
```

Comment:
is that property set to a different value anywhere yet?

Reply:
Yup, this is set to false already on one of our experiments in the runtimelab repo and it will be added to the rest as well:
https://github.com/dotnet/runtimelab/blob/Utf8String/src/libraries/libraries-packages.proj#L7

Also the idea of this property is that it would be used once we turn up the crank for servicing as the way to disable all packages build that are not specifically marked as build.